docs/hpc/R.md

```diff
@@ -29,10 +29,10 @@ If your script requires a package that is not listed here, a [new container](htt
 To run a script in the R programming language using the login node, log in to the [Panther Shell Access](https://ndclab.github.io/wiki/docs/hpc/accessing.html#login-node). Use `cd` to navigate to the folder that contains your script. From that location, run the following command:
 
 ```
-sh /home/data/NDClab/tools/lab-devOps/scripts/R/rrun.sh <your-script-name>.R
+bash /home/data/NDClab/tools/lab-devOps/scripts/R/rrun.sh <your-script-name>.R [# nodes] [hours of walltime requested]
```

Comment:
Apparently it probably doesn't matter, sh is symlinked to /bin/bash on the HPC. If it was symlinked to another shell in the future it would be important to make sure we're calling bash for these scripts.

Reply:
And I just thought it might be helpful to be able to specify the nodes needed and walltime needed with the command, sure I can elaborate a bit more on that.